sktime/forecasting/model_evaluation/_functions.py

```diff
@@ -320,23 +285,67 @@ def evaluate(
         Row index is splitter index of train/test fold in `cv`.
         Entries in the i-th row are for the i-th train/test split in `cv`.
         Columns are as follows:
+
         - test_{scoring.name}: (float) Model performance score. If `scoring` is a list,
-            then there is a column withname `test_{scoring.name}` for each scorer.
+        then there is a column withname `test_{scoring.name}` for each scorer.
+
         - fit_time: (float) Time in sec for `fit` or `update` on train fold.
         - pred_time: (float) Time in sec to `predict` from fitted estimator.
         - len_train_window: (int) Length of train window.
         - cutoff: (int, pd.Timestamp, pd.Period) cutoff = last time index in train fold.
         - y_train: (pd.Series) only present if see `return_data=True`
-          train fold of the i-th split in `cv`, used to fit/update the forecaster.
+        train fold of the i-th split in `cv`, used to fit/update the forecaster.
+
         - y_pred: (pd.Series) present if see `return_data=True`
-          forecasts from fitted forecaster for the i-th test fold indices of `cv`.
+        forecasts from fitted forecaster for the i-th test fold indices of `cv`.
+
         - y_test: (pd.Series) present if see `return_data=True`
-          testing fold of the i-th split in `cv`, used to compute the metric.
+        testing fold of the i-th split in `cv`, used to compute the metric.
+
+    Notes
+    -----
+    The experiment run is the following:
```

Comment:
Personally, I would leave this in the preamble. Why move this down?

Reply:
Resolved here https://github.com/sktime/sktime/pull/5065/commits/2e31622058ae29448613333495b6d9143d41201a